versions/1/manifest.spec.yml

```diff
@@ -216,10 +271,6 @@ spec:
         - name
         - title
         - description
-    datasets:
-      type: array
-      items:
-        $ref: "./dataset/manifest.spec.yml" 
```

Comment:
Hmm, I wonder why this was there 🤔

Reply:
I have no idea, I didn't find any references.